joomla/joomla-cms
# [4.1] Simplify Requirements for Task Routine XMLs

Pull Request for Issue #36449.

### Summary of Changes
- Fixes form for the Image Size Check task routine and improves a language string for clarity.
- Removes requirement of the "showFront" attribute for the `task_params` fieldset.

### Testing Instructions
- Admin > System > Scheduled Tasks
- Click "Create Task" or similar
- Select "Image Size Check" routine
- Fill in the form and save


### Actual result BEFORE applying this Pull Request
- Form fails to submit due to validation error (missing required data).
- Routine XMLs require a `showFront="true"` attribute.

### Expected result AFTER applying this Pull Request
- Form is submitted, task is created.
- Routine XMLs no longer require require the attribute to work correctly.

### Documentation Changes Required
\-
